charts/argo-cd/templates/aggregate-roles.yaml

```diff
@@ -14,9 +14,6 @@ rules:
   {{- if .Values.applicationSet.enabled }}
   - applicationsets
   {{- end }}
-  {{- if .Values.server.extensions.enabled }}
-  - argocdextensions
-  {{- end }}
```

Comment:
argocdextensions  CRD is removed ♻️

Reply:
to be honest, it should be a breaking release of the chart

Reply:
Yes, so I bumped minor version as following [Versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning) and put changelog in README.
Do you want us to do more things ?